### PR TITLE
Adds new Dead Space inspired Advanced MODsuit Theme

### PR DESCRIPTION
**Pull request by dawsonkeyes**
## About The Pull Request

Adds a new theme for the Chief Engineer's Advanced MODsuit, in the style of the Advanced RIG from Dead Space 2.
Lovingly sprited by Triiodine#7435

image

## Why It's Good For The Game

At the moment, only the ERT and Medical MODsuits currently get paint kit reskins. This at least adds more use to such a system, and adds a hell of a lot more possible flavor!
Plus, SS13 engineer suits have always been Dead Space inspired. It's a return to form!

## Changelog

:cl:
add: The Advanced MODsuit now has a new style in the theme of Dead Space!
/:cl:

### CODE REVIEW

**Comment by dawsonkeyes** [comment]
feature freeze got me like d r a f t

**Comment by dawsonkeyes** [comment]
oh right feature freeze

**Comment by dawsonkeyes** [comment]
will reopen this like a few days post-freeze dont worry people

**Comment by Dawson1917** [comment]
I don't think departmental/head suits should have any selectable themes in the first place but if they do they should probably share the same colour scheme as their department/head of staff. This sprite would be well-suited for a unique modsuit findable somewhere else in the game, like the other dead space-inspired suit from Charlie station

**Comment by dawsonkeyes** [comment]
> I don't think departmental/head suits should have any selectable themes in the first place but if they do they should probably share the same colour scheme as their department/head of staff. This sprite would be well-suited for a unique modsuit findable somewhere else in the game, like the other dead space-inspired suit from Charlie station

No

**Review by Fikou — CHANGES_REQUESTED** [review]
the helmet flags need to reflect the sprite, yours act like the helmet doesnt cover almost anything

the boot and gauntlet sprites dont fit the rest of the suits

the way chestplate and backpack is split on the mob sprites is dumb and the charge meter should not be on the chest

most suits follow one palette of bluish greys, could you test if these colors work good on this suit?

**Comment by dawsonkeyes** [comment]
> the helmet flags need to reflect the sprite, yours act like the helmet doesnt cover almost anything
> 
> the boot and gauntlet sprites dont fit the rest of the suits
> 
> the way chestplate and backpack is split on the mob sprites is dumb and the charge meter should not be on the chest
> 
> most suits follow one palette of bluish greys, could you test if these colors work good on this suit?

sprite changes i can do, but what do you mean by the helmet flags?

File: code/modules/mod/mod_theme.dm [hunk 1]
```diff
@@ -195,6 +195,29 @@
...
 			),
 		),
+		"titan" = list(
+			HELMET_LAYER = NECK_LAYER,
+			HELMET_FLAGS = list(
+				UNSEALED_CLOTHING = SNUG_FIT,
+				SEALED_CLOTHING = THICKMATERIAL|STOPSPRESSUREDAMAGE,
+				UNSEALED_INVISIBILITY = HIDEFACIALHAIR,
+				SEALED_INVISIBILITY = HIDEMASK|HIDEEARS|HIDEEYES|HIDEFACE|HIDEHAIR|HIDESNOUT,
+				SEALED_COVER = HEADCOVERSMOUTH|HEADCOVERSEYES|PEPPERPROOF,
```
Comment (Fikou) [inline code/modules/mod/mod_theme.dm:205]: this part, the helmet seems to be a helmet that fully extends over the head even when unsealed, so the flags/layer should reflect that

**Comment by dawsonkeyes** [comment]
fuck it I’ll fix this later to spec and pr it then
damn merge conflicts kickin my ass

**Pull request closed without merging**